etc/schema.json

```diff
@@ -3615,6 +3615,9 @@
                         "password_message": {
                             "type": "string"
                         },
+                        "password_redacted": {
+                            "type": "boolean"
```

Comment:
When I notice -- all new fields should have a short description.

Reply:
🙇🏽‍♀️  slipped by >__<'